### DISCUSSION TITLE
Hamlib and TCPIP

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#91] [Show and tell]
Hi,

here is an info from Martin, OK1RR about the use of IP addresses in Hamlib.
https://www.cqrlog.com/

**Comment by foldynl** [comment]
Thank you for the link. I also read it. There are few important things related to QLog in the first sentence 

> Hamlib, at least the **version 4.5** has a **not well documented** option 

- Version 4.5 is development version, which is not used by QLog. I guess that this version is not present in any distro now.
- "not well documented" that a problem of Hamlib. If it is not well documented, it may mean that it may change.

But it's good to watch.

**Comment by dl2ki** [comment]
Yes, you can observe this information once. Martin uses a 'K4' and may be able to use this feature. 
There are also other rigs that can be addressed via the IP address. 

If you need this or which advantages this has, I can't judge. Du kannst ja bei Martin einmal direkt anfragen, welche Vorteile er dadurch sieht.

**Comment by foldynl** [comment]
It would be a great honor for me if Martin, as co-author of CQRLog, use QLog. He's  a CW-enthusiastic, and  QLog doesn't have any CW support (yet). 

But now, seriously. It's an interesting news, but 4.5 will be used it will be ready and stable and present in Linux distros. And I see another obstacle in the fact that I cannot test it - I don't have K4 or any IP-direct-connected rig.
